selectors.yml

```diff
@@ -3,7 +3,13 @@ selectors:
     definition:
       union:
         - method: package
-          value: livequery_models
+          value: livequery_base
         - exclude:
           - livequery_models.deploy.marketplace.quicknode.test_quicknode_solana*
-          - livequery_models.deploy.marketplace.quicknode.test_quicknode_polygon*
\ No newline at end of file
+          - livequery_models.deploy.marketplace.quicknode.test_quicknode_polygon*
+          - livequery_models.deploy.marketplace.bitquery.* # API Credits not enough
+          - livequery_models.deploy.marketplace.covalent.* # API Credits not enough
+          - livequery_models.deploy.marketplace.coingecko.* # API Credits not enough
+          - livequery_models.deploy.marketplace.allday.* # GraphQL API not working
+          - livequery_models.deploy.marketplace.playgrounds.* # API Endpoints not working
+          - livequery_models.deploy.marketplace.strangelove.* # API Endpoints not working
```

Comment:
i added a exclude for apilayer yesterday. should reflect it here too to avoid error on pull_request test_udf

Reply:
This PR is being closed in favor of https://github.com/FlipsideCrypto/livequery-models/pull/114

Reply:
Details in this [thread](https://flipsidecrypto.slack.com/archives/C03168ANDGE/p1740459872655789?thread_ts=1740459190.407489&cid=C03168ANDGE)